thmsndk/Screeps3D
# fix: flags will be removed from the client if a flag no longer exist.

Fixed Stale Flags, and flags not getting removed isse:
Fixes https://github.com/thmsndk/Screeps3D/issues/109

To Test:
1. Load the Web/Steam Client
1. Load Screeps3D with this Branch
1. On the Web/Steam Client Spam a bunch of flags in a Room
1. While watching the Screeps3D client, remove the flags on the Steam/Web client and the flags should also disappear on the Screeps3d Client.
